collections/message.js

```diff
@@ -0,0 +1,9 @@
+const mongoose = require('mongoose');
+
+const Message = mongoose.model('Message', {
```

Comment:
Does this require a `mongoose.Schema` to be passed to the `Mongoose.model`?

http://mongoosejs.com/docs/models.html